Edit UI Canvas components and placement procedure (#1852)
o3de/o3de.org: content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md

| **Is pixel aligned** | Makes textures look sharper by rounding the position of the elements' corners to the nearest exact pixel. For example, if the position of a corner of an element rectangle is at (123.45, 678.90), then it is rounded to (123.00, 679.00). | Boolean | `Enabled` |
| **Is text pixel aligned** | Makes text look crisper by rounding text positions down to the nearest pixel. An exception to this rule occurs when fonts have been scaled down, in which case the text position is rounded to the nearest pixel. If the property is not checked, the text position is not rounded. You might consider disabling this property if, for example, a text element will animate or move. | Boolean | `Enabled` |
| **Render to texture** | If enabled, the canvas is drawn to a material's **Base Color** texture rather than to the screen. You must select an Attachment Image asset to use for the texture in the **Render Target** property. For more information, refer to [Placing UI Canvases in the 3D World](/docs/user-guide/interactivity/user-interface/canvases/placing-canvases-3d). | Boolean | `Disabled` |

> | **Render Target** | Selects the Attachment Image asset to render this canvas to. To generate an Attachment Image asset, create a new source file with the `.attimage` extension, refer to the example `.attimage` source file below. <br> <br>*This property is available only if **Render to texture** is set to `Enabled`.* | Attachment Image asset | None |

Reviewer: Style guideline: Avoid using "above" and "below", since pagination might change this.
```suggestion
| **Render Target** | Selects the Attachment Image asset to render this canvas to. To generate an Attachment Image asset, create a new source file with the `.attimage` extension. Refer to the `.attimage` source file example that follows. <br> <br>*This property is available only if **Render to texture** is set to `Enabled`.* | Attachment Image asset | None |
```

}
```

For more detail about the AttachmentImageAsset class, refer to the following references:

> - [AttachmentImageAsset class reference](/docs/api/gems/atom/class_a_z_1_1_r_p_i_1_1_attachment_image_asset.html)

Reviewer 2: ```suggestion
- [`AttachmentImageAsset` class API reference](/docs/api/gems/atom/class_a_z_1_1_r_p_i_1_1_attachment_image_asset.html)
```

> - [ImageDescriptor struct reference](/docs/api/gems/atom/struct_a_z_1_1_r_h_i_1_1_image_descriptor.html)

Reviewer 2: ```suggestion
- [`ImageDescriptor` struct API reference](/docs/api/gems/atom/struct_a_z_1_1_r_h_i_1_1_image_descriptor.html)
```

> - [DXGI Format enum definitions](https://github.com/o3de/o3de/blob/38261d08007a1ad14135cf790cb855ccca4d8595/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/AzDXGIFormat.h)

Reviewer 2: ```suggestion
- [`DXGI_Format` enum definitions](https://github.com/o3de/o3de/blob/38261d08007a1ad14135cf790cb855ccca4d8595/Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/AzDXGIFormat.h)
```
